.fixtures.yml

```diff
@@ -14,12 +14,12 @@ fixtures:
       ref: "1.0.0"
   repositories:
     "stdlib":
-      "repo": "git://github.com/puppetlabs/puppetlabs-stdlib.git"
+      "repo": "git@github.com:puppetlabs/puppetlabs-stdlib.git"
     "translate":
-      "repo": "https://github.com/puppetlabs/puppetlabs-translate"
-    facts: 'git://github.com/puppetlabs/puppetlabs-facts.git'
-    puppet_agent: 'git://github.com/puppetlabs/puppetlabs-puppet_agent.git'
+      "repo": "git@github.com:puppetlabs-toy-chest/puppetlabs-translate.git"
```

Comment:
This one should auto-redirect if we leave it as `https://`.

Reply:
I have switched to the https version as per your recommendation. see file difference.